app/assets/stylesheets/custom.scss

```diff
@@ -30,6 +30,8 @@ h3 {
 }
 
 .main-body {
+  margin-bottom: -142px;
+  min-height: 70vh;
```

Comment:
Properties should be ordered margin-bottom, margin-bottom, margin-top, min-height